tests/integration/Core/Checkout/Cart/Order/RecalculationServiceTest.php

```diff
@@ -324,35 +324,6 @@ public function testRecalculationControllerWithNonSystemLanguage(): void
         static::assertEquals($this->getDeDeLanguageId(), $order->getLanguageId());
     }
 
-    public function testFetchOrder(): void
```

Comment:
@mstegmeyer @cyl3x  I removed this test because of 3 reasons:

1. It's testing a private function, which I don't think is needed
2. The code is covered by other tests (i.e. every test that fetches an `Order`)
3. The `fetchOrder` method changed to do everything required to fetch a valid order

Please just confirm you are happy with this change